### PR TITLE
Create front end cypress tests

### PR DESCRIPTION
**Pull request by laoneo**
### Summary of Changes
First shot of front end cypress tests. Target is to have for every view an automated test.

### CODE REVIEW

File: tests/cypress/integration/site/components/com_content/Blog.cy.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
+
+beforeEach(function() {
+  cy.task('queryDB', 'DELETE FROM #__content');
```
Comment (HLeithner) [inline tests/cypress/integration/site/components/com_content/Blog.cy.js]: What's with the asset table? wouldn't this be "broken"

**Comment by HLeithner** [comment]
other question, wouldn't it make more sense to use the webservices api to create articles instead of doing database queries directly?

File: tests/cypress/integration/site/components/com_content/Blog.cy.js (hunk 1 above)
Comment (laoneo) [reply tests/cypress/integration/site/components/com_content/Blog.cy.js]: Nope, there are just empty references, which can be ignored as we are in a test environment. It is much more important to start on an empty state.

**Comment by laoneo** [comment]
Like that it is much faster.

**Pull request merged**